diff --git CMakeLists.txt CMakeLists.txt
index dd71330..b5a835b 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -58,7 +58,7 @@ find_package(Eigen3 REQUIRED)
 include_directories(SYSTEM ${EIGEN3_INCLUDE_DIR})
 
 # add OSQP library
-find_package(osqp REQUIRED)
+#find_package(osqp REQUIRED)
 
 set(LIBRARY_TARGET_NAME OsqpEigen)
 
@@ -83,7 +83,7 @@ add_library(${LIBRARY_TARGET_NAME} ${${LIBRARY_TARGET_NAME}_SRC} ${${LIBRARY_TAR
 target_include_directories(${LIBRARY_TARGET_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
   "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>")
 
-target_link_libraries(${LIBRARY_TARGET_NAME} PRIVATE osqp::osqp Eigen3::Eigen)
+target_link_libraries(${LIBRARY_TARGET_NAME} PRIVATE osqp Eigen3::Eigen)
 
 set_target_properties(${LIBRARY_TARGET_NAME} PROPERTIES
   VERSION ${PROJECT_VERSION}
